pyproject.toml

```diff
@@ -6,11 +6,11 @@ authors =["Sergey Putko <psvmcc+ansible-galaxy@gmail.com>", "George Shuklin <geo
 license = "BSD/MIT"
 
 [tool.poetry.dependencies]
-python = "^3.10"
-ansible-core = "^2.14.3"
+python = ">=3.9, <3.12"
+ansible-core = "2.15"
```

Comment:
Perhaps we want here `ansible-core = "<= 2.16"`?

Reply:
Thanks.